aiidalab_widgets_base/computational_resources.py

```diff
@@ -858,6 +858,17 @@ def _configure_computer(self, computer: orm.Computer):
 
         return True
 
+    def _configure_computer_local(self, computer: orm.Computer):
+        """Configure the computer with local transport"""
+        authparams = {
+            "use_login_shell": self.use_login_shell.value,
+            "safe_interval": self.safe_interval.value,
+        }
+        user = orm.User.collection.get_default()
+        computer.configure(user=user, **authparams)
```

Comment:
Note that we need to catch any potential errors here. Since this was not done in the original function, I am keeping that for separate PR.